packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -59,6 +60,11 @@ const Redirect = ({
             }
             break;
         }
+        case 'document_verification': {
+            sessionStorage.setItem('redirect_url', routes.proof_of_identity);
+            window.location.href = Login.loginUrl();
```

Comment:
shouldn't we check if client is already logged in?

Reply:
No, we want to ensure the user logs in again so that the account they're using to supply their documents for is correct.